tails_files/install.sh

```diff
@@ -21,6 +21,13 @@ DESKTOP=$HOMEDIR/Desktop
 ANSIBLE=$PERSISTENT/securedrop/install_files/ansible-base
 SSH_ALIASES=false
 
+# check for Tails 2.x
+source /etc/os-release
```

Comment:
> Expected the answer to be that grepping with set -e can be complicated, since grep by default exits non-zero if there wasn't a match.

@conorsch Remember that `set -e` does **not** always mean the script will exit if a command returns non-zero. For example, it will not exit if a command returns non-zero in a conditional. Run `info bash` and search for `errexit` for more info.